packages/rn-tester-e2e/package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "webdriverio-tests",
+    "version": "0.1.0",
+    "description": "",
+    "private": true,
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "devDependencies": {
+        "@babel/core": "^7.19.6",
+        "@babel/eslint-parser": "^7.19.1",
+        "@babel/runtime": "^7.12.5",
+        "@react-native-community/eslint-config": "^2.0.0",
+        "appium": "^2.0.0-beta.40",
```

Comment:
Not seeing reflected in yarn lockfile. Could we yarn install (also so we can see how many deps added)